pocs/tests/test_rs232.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+import serial  # PySerial, from https://github.com/pyserial/pyserial
+
+from pocs.utils import rs232
+from pocs.utils.config import load_config
+
+from pocs.tests.serial_handlers import protocol_buffers
+from pocs.tests.serial_handlers import protocol_no_op
+from pocs.tests.serial_handlers import protocol_test
+
+THE_HOOKS = None
+
+
+def create_hooks(ser, *args, **kwargs):
+    THE_HOOKS = SimpleSerialHooks(ser)
+
+
+@pytest.fixture(scope='function')
+def handler():
+    # Install our test handlers for the duration
+    serial.protocol_handler_packages.append('pocs.tests.serial_handlers')
+    yield True
+    # Remove our test handlers
+    serial.protocol_handler_packages.remove('pocs.tests.serial_handlers')
+
+
+def test_basic(handler):
+    # Confirm we can create the SerialData object.
+    ser = rs232.SerialData(port='no_op://', open_delay=0)
+    # Peek inside, it should have a PySerial instance as member ser.
+    assert ser.ser
+    assert ser.ser.__class__.__name__ == 'NoOpSerial'
+    print(str(ser.ser))
+    # Open is automatically called by SerialData.
+    assert ser.is_connected
+    # Not using threading.
```

Comment:
`threaded=True` is the default on `SerialData`. Does this get unset in one of the serial_handler files above?

Reply:
My mistake, I misunderstood.